spec/integration/fetching_linked_items_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe "Fetching linked items", type: :request do
+  describe "GET /incoming-links/:base_path_without_root" do
+    it "returns the items linked to an item" do
+      item = create(:content_item, :with_content_id)
+      create(:content_item, content_id: 'ID-1', base_path: '/a', title: "A", links: { "parent" => [item.content_id] })
+      create(:content_item, content_id: 'ID-2', base_path: '/b', title: "B", links: { "parent" => [item.content_id] })
+
+      get "/incoming-links#{item.base_path}?types[]=parent&types[]=topics"
+
+      expect(parsed_response["parent"]).to eql([
+        {
+          "content_id" => "ID-1",
+          "title" => "A",
+          "base_path" => "/a",
+          "description" => nil,
+          "api_url" => "http://www.example.com/content/a",
+          "web_url" => "https://www.test.gov.uk/a",
+          "locale"=>"en"
+        },
+        {
+          "content_id" => "ID-2",
+          "title" => "B",
+          "base_path" => "/b",
+          "description" => nil,
+          "api_url" => "http://www.example.com/content/b",
+          "web_url" => "https://www.test.gov.uk/b",
+          "locale" => "en"
+        },
+      ])
+
+      expect(parsed_response["topics"]).to eql([])
```

Comment:
No, that would mean that if `parsed_response["topics"]` returned a hash, this test would also pass. `{}.empty?` is also true.